app/src/main/java/com/github/se/studybuddies/navigation/NavigationActions.kt

```diff
@@ -16,4 +16,8 @@ class NavigationActions(private val navController: NavHostController) {
   fun goBack() {
     navController.popBackStack()
   }
+
+  companion object {
+    fun navigateTo(route: String) {}
+  }
```

Comment:
We already have a navigateTo function. Why do you need to create a new one ?

Reply:
i encountered error when it had no "companion object" version, I dont really understnad why but this was the recommanded solution on androidstudio alt enter